### PR TITLE
Apply boa-sdk-ts to Stoa

### PR DESCRIPTION
**Pull request by MichaelKim20**
I copied all the data modules and utilities Stoa had on the SDK. (https://github.com/bpfkorea/boa-sdk-ts/pull/54)
This installed the branch of https://github.com/bpfkorea/boa-sdk-ts/pull/54 as an npm module and modified it to use SDK.
1. I removed folders /src/modules/data and /src/modules/utils.
2. I changed the reference to the library to `boa-sdk-ts`.
3. I have modified the compatibility with the sodium library.
If https://github.com/bpfkorea/boa-sdk-ts/pull/54 is merged first, it will be able to complete.

Relates to #180

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,23 +23,20 @@
     "@ctrl/ts-base32": "^1.2.4",
     "@types/app-root-path": "^1.2.4",
     "@types/cors": "^2.8.8",
-    "@types/sodium-native": "^2.3.5",
     "@types/sqlite3": "^3.1.6",
-    "ajv": "^6.12.6",
     "argparse": "^2.0.1",
     "assert": "^2.0.0",
     "axios": "^0.19.2",
+    "boa-sdk-ts": "https://github.com/MichaelKim20/boa-sdk-ts.git",
```
Comment (MichaelKim20) [inline package.json]: If https://github.com/bpfkorea/boa-sdk-ts/pull/54 is merged first, I will modify this.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -20,6 +20,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
+    - run: npm install
```
Comment (Geod24) [inline .github/workflows/main.yml:23]: Why the change ?

File: package.json (hunk 1 above)
Comment (Geod24) [reply package.json]: It will need to

File: .github/workflows/main.yml (hunk 2 above)
Comment (MichaelKim20) [reply .github/workflows/main.yml:23]: The npm install script inside the boa-sdk-ts did not run automatically when it was 'npm ci'.

**Comment by MichaelKim20** [comment]
I created a package-lock.json using Node version 15.3 and was able to pass through CI

File: package.json (hunk 1 above)
Comment (MichaelKim20) [reply package.json]: I changed to "boa-sdk-ts": "git+https://github.com/bpfkorea/boa-sdk-ts.git"

File: src/main.ts [hunk 3]
```diff
@@ -36,23 +37,29 @@ const stoa: Stoa = new Stoa(config.database.filename,
...
+        logger.error(`Failed to load Sodium library: ${error}`);
         process.exit(1);
-    })
-}).catch((error) => {
+    });
+})
+.catch((error:any) => {
     logger.error(`Failed to create LedgerStorage: ${error}`);
     process.exit(1);
 });
```
Comment (Geod24) [inline src/main.ts]: Instead of nesting, please chain

Comment (MichaelKim20) [reply src/main.ts]: I changed it.
